File: pall01.java
import java.util.*;

class pall01 extends print{
    public static void main(String[] args){
        Scanner in=new Scanner(System.in);
        int t=in.nextInt();
        while(t-->0){
            int n=in.nextInt();
            int m=reverse(n);
            if(n==m){
                print("wins");
            }
            else print("loses");
        }
    }

    private static int reverse(int x){
        int a=x,b=0;
        while(a>0){
            int temp=a%10;
            b=b*10+temp;
            a/=10;
        }
        return b;
    }
}

File: pall01.py
def reverse(x):
    b=0
    while(x>0):
        temp=int(x%10)
        b=int(b*10+temp)
        print(b,temp,x)
        x=int(x/10)
    print(b)
    return(b)


t=int(input())
for i in range(t):
    n=int(input())
    m=int(reverse(n))
    print(("loses","wins") [m==n])
